/gazebo_ros_control:   
  pid_gains:
    shoulder_pan_joint:
      p: 1000.0
      i: 0.00
      d: 10.0
    shoulder_lift_joint:
      p: 1000.0
      i: 0.00
      d: 10.0
    elbow_joint:
      p: 1000.0
      i: 0.00
      d: 10.0
    wrist_1_joint:
      p: 1000.0
      i: 0.00
      d: 10.0

    wrist_2_joint:
      p: 1000.0
      i: 0.00
      d: 10.0
    wrist_3_joint:
      p: 1000.0
      i: 0.00
      d: 10.0
